### DISCUSSION TITLE
Welcome to Scaffold-Eth 🏗  Challenges Discussions!

### DISCUSSION THREAD

**Discussion by codenamejason** [scaffold-eth/scaffold-eth-challenges#5] [Announcements]
## 👋  Welcome Developers!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about the challenges.
  * Share ideas about challenges you faced and how you got past them.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pdragnev** [comment]
Typing command yarn chain produces this error:
![image](https://user-images.githubusercontent.com/44343591/224508670-ee2a7403-2a9f-47ba-9e3e-2b9ee6ea55fb.png)
any suggestions?
